.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: 'close stale issues/PRs'
+on:
+  schedule:
+    - cron: '* */6 * * *'
+  workflow_dispatch:
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@87c2b794b9b47a9bec68ae03c01aeb572ffebdb1
+        with:
+          repo-token: ${{ github.token }}
+          days-before-stale: 21
+          days-before-close: 7
+          only-labels: ""
+          operations-per-run: 100
+          remove-stale-when-updated: false
+          debug-only: false
+          ascending: false
+
+          exempt-issue-labels: "Triage: Accepted"
+          stale-issue-label: "Triage: Stale"
+          stale-issue-message: "This message is required, but you shouldn't see it."
+          skip-stale-issue-message: true
+          close-issue-label: ""
+          close-issue-message: ""
```

Comment:
Shouldn't we have some sort of a message to explain what happened?

Reply:
I was thinking the default message GitHub adds would be enough to start with, but I agree a friendlier comment is a better user experience. Here's what I came up with:

<img width="899" alt="Screen Shot 2021-01-04 at 3 52 35 PM" src="https://user-images.githubusercontent.com/134455/103578660-f65fab00-4ea4-11eb-96d3-eb83887bc025.png">